### ISSUE TITLE
Allow configuring the location of package.json 

### ISSUE THREAD

**Issue by MHarris021** [srs/gradle-node-plugin#107]
Currently the plugin assumes that package.json will be located at the root of the project directory. It would be helpful to have the node configuration block be able to change this default location.

**Comment by a3rnett** [comment]
+1 It seems that the built in npmInstall honors the nodeModulesDir setting as the location for the package.json file, however other npm commands (eg 'npm install' or 'npm run foo') do not accomodate.
And in the interim, is there a workaround?

**Comment by raulgd** [comment]
In the source code, I checked and there is a workingDir property, use something of the like:

```groovy
task appNpmInstall(type: NpmTask) {
	description = "Installs dependencies from package.json"
	workingDir = file("${project.projectDir}/src/main/web")
	args = ['install']
}
```

**Comment by srs** [comment]
This is now fixed. One side effect (for now) is that ``workingDir`` property cannot be set. This will be fixed in #117.
